frontend/.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
-  "typescript.tsdk": "node_modules/.pnpm/typescript@4.9.5/node_modules/typescript/lib",
+  "typescript.tsdk": "node_modules\\typescript\\lib",
```

Comment:
Could we revert this change? We don't need this and checking out locally just overwrites this with the old value

Reply:
Hi @puckzxz, apologies for the late response.

I've made the changes you requested (removed package-lock & reverted changes on the the typescript.tsdk file). Please confirm🙏🏾